public/components/GeoPlot.vue

```diff
@@ -232,8 +234,32 @@ export default Vue.extend({
     mapZoom(): number {
       return routeGetters.getGeoZoom(this.$store);
     },
+
     rowSelection(): RowSelection {
       return routeGetters.getDecodedRowSelection(this.$store);
+    },
+
+    isSatelliteImages(): boolean {
+      return datasetGetters.isSatelliteImages(this.$store);
+    },
+
+    currentSatelliteBand(): SatelliteBand {
+      return appGetters.getCurrentSatelliteBand(this.$store);
```

Comment:
@cdbethune said that the satellite band will be a "root" variable define by the user, this is why I add it to the `store/app`.

Reply:
I think we got signals crossed on "root" vs. "route".  What you will have access to in the store is a stucture similar to:

```typescript
interface BandMapping {
    mappingID: string, // needs to be unique
    displayName: string, // exact string to display in UI
    mapping: int[] // length of 3 containing band mapping for r, g, b in order
}
```
and we'll have to store the active `mappingID` in the route (store/route) so that we have it on refresh.